### DISCUSSION TITLE
Using google bard instead of chat GPT

### DISCUSSION THREAD

**Discussion by elsayed07** [morpheuslord/GPT_Vuln-analyzer#32] [Ideas]
Hi there,

I was wondering what it would take to replace ChatGPT with Google Bard in this project.
Would you be interested in doing it? Great work, by the way!

**Comment by morpheuslord** [comment]
Ya, I am interested to do that but Bard has no API support as of now. If there is a API support the.
Then this would be possible maybe a choice system for bard or openai

**Reply by elsayed07** [reply]
there is an unofficial package for Bard API: https://github.com/dsdanielpark/Bard-API

**Reply by morpheuslord** [reply]
Thats great will work on this

**Reply by morpheuslord** [reply]
I tried using this and the support and usage are complicated. I am waiting for an official release for it to work reliably.

**Comment by morpheuslord** [comment]
Thanks for the advise 👍

**Comment by morpheuslord** [comment]
@elsayed07  Bro You can check the latest release I have added bard into the mix thanks for the suggestion 👍

**Reply by elsayed07** [reply]
Bro you're doing a great job Keep it going
How much time have you waited to get access to PaLM API?

**Reply by morpheuslord** [reply]
2 hours maybe. But for some, it might take 1 to 2 days.
